app/components/modals/media-reaction.js

```diff
@@ -44,7 +44,8 @@ export default Component.extend({
     const libraryEntry = get(this, 'libraryEntry');
     let reaction = null;
     if (!get(this, 'createOnly')) {
-      reaction = yield libraryEntry.belongsTo('mediaReaction').load();
+      reaction = yield get(this, 'libraryEntry.mediaReaction') ||
```

Comment:
Load should just resolve the value if we already have it loaded, but if we need to check it for some reason:

Check

`libraryEntry.belongsTo('mediaReaction').value() !== null`